docs/get-started/faq.mdx

```diff
@@ -5,6 +5,11 @@ slug: /faq
 sidebar_position: 100
 ---
 
+import { Tag } from "@site/src/components/Tag";
+
+import githubSwitchAccountSettings from "@site/static/img/github-switch-account-settings.png";
```

Comment:
Could you please move images from static folder, static is to serve static files, not imported files.

Reply:
Also optimized all images using ImageOptim.